plugins/inputs/logparser/logparser.go

```diff
@@ -286,6 +312,16 @@ func (l *LogParserPlugin) Stop() {
 	defer l.Unlock()
 
 	for _, t := range l.tailers {
+		if !l.FromBeginning {
+			// store offset for resume
+			offset, err := t.Tell()
+			if err == nil {
+				offsets[t.Filename] = offset
```

Comment:
Doesn't `offsetsMutex` need to lock for this?

Reply:
Yes, poor copy from plugins/tail